src/EventStore.Client/HttpFallback.cs

```diff
@@ -17,11 +17,16 @@ internal class HttpFallback : IDisposable {
             _defaultCredentials = settings.DefaultCredentials;
 			
 			var handler = new HttpClientHandler();
-			if (!settings.ConnectivitySettings.Insecure && !settings.ConnectivitySettings.TlsVerifyCert) {
+			if (!settings.ConnectivitySettings.Insecure) {
 				handler.ClientCertificateOptions = ClientCertificateOption.Manual;
-				handler.ServerCertificateCustomValidationCallback = (_, _, _, _) => true;
+
+				if (settings.ConnectivitySettings.TlsCaFile != null)
+					handler.ClientCertificates.Add(settings.ConnectivitySettings.TlsCaFile);
+
+				if (!settings.ConnectivitySettings.TlsVerifyCert)
+					handler.ServerCertificateCustomValidationCallback = (_, _, _, _) => true;
```

Comment:
minor quibble. Could be

```
handler.ServerCertificateCustomValidationCallback = delegate { return  true; };
```
since we do not care about the parameters but the functionality is the same.